librz/util/bitvector.c

```diff
@@ -929,7 +929,7 @@ RZ_API bool rz_bv_complement_1_inplace(RZ_INOUT RZ_NONNULL RzBitVector *bv) {
 		return true;
 	}
 
-	if (!(bv->bits.large_a && bv->bits.large_a)) {
+	if (!bv->bits.large_a) {
```

Comment:
@Rot127 this patch seems wrong, please have a look.

Reply:
The only thing it was supposed to do was checking the `large_a` is not NULL.